plugins/GrapesJsBuilderBundle/Translations/de/javascript.ini

```
@@ -1,12 +1,12 @@
-grapesjsbuilder.sourceEditBtnLabel="Bearbeiten"
+grapesjsbuilder.sourceEditBtnLabel="Speichern"
 grapesjsbuilder.sourceCancelBtnLabel="Abbrechen"
 grapesjsbuilder.sourceEditModalTitle="Code bearbeiten"
 grapesjsbuilder.sourceSyntaxError="Bitte korrigiere den folgenden Fehler:"
-grapesjsbuilder.deleteAssetConfirmText="Möchten Sie diese Datei wirklich löschen?"
+grapesjsbuilder.deleteAssetConfirmText="Möchtest du diese Datei wirklich löschen?"
```

Just making @ekkeguembel's suggestion visible and it can be committed directly from the button bellow.
```suggestion
grapesjsbuilder.deleteAssetConfirmText="Möchtest Sie diese Datei wirklich löschen?"
```

---

@escopecz not quite: It is "Möchten Sie", not "Möchtest Sie" :)

---

Thanks, updated